server/src/main/java/org/elasticsearch/search/query/QuerySearchResult.java

```diff
@@ -245,10 +246,6 @@ public void releaseAggs() {
         }
     }
 
-    private void close() {
```

Comment:
This was a weird redundant method, I mainly figured this was easier to reason about GC wise when doing it inline in a lambda, but also no point in having this method when `javac` generates it for us :)